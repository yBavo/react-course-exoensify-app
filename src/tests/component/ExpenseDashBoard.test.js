import React from 'react'
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ExpenseDashBoardPage from '../../component/ExpenseDashBoard'


configure({ adapter: new Adapter() });

test('Should render ExpenseDashBoardPage', () => {
  const wrapper = shallow(<ExpenseDashBoardPage />);
  expect(wrapper).toMatchSnapshot();
})