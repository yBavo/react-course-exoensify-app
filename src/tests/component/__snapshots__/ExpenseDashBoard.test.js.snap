// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseDashBoardPage 1`] = `
<div>
  <h1>
    Expense DashBoard
  </h1>
  <div>
    <Connect(ExpenseListFilters) />
    <Connect(ExpenseList) />
  </div>
</div>
`;
